README.md

```diff
@@ -116,5 +116,6 @@ Static code analysis
 In order for a PR to be accepted the following pylint command needs to result in 10/10 analysis:
 ```
 cd MITuna/tuna
+pylint -f parseable -d duplicate-code --max-args=8 --indent-string '  ' *.py
```

Comment:
Wouldn't the next line do this?

Reply:
I think this accidentally flagged as an edit...removed manual processes

Reply:
done!